src/main/webapp/stylesheets/teammatesCommon.css

```diff
@@ -712,10 +712,14 @@ Prevent Long URL From Breaking Out of Container
     hyphens: auto;
 }
 
+/* highlight search keywords in result on Admin search page */
```

Comment:
The style definition should not mention where it is applied. It's just a style that can be used anywhere in the app. Simply explain what it does. If at all, can give an example of where it can be used.